components/Questionnaire/Two/index.vue

```diff
@@ -0,0 +1,253 @@
+                        height="88"
+                        :src="files.pelatihan.source"
+                        alt="Foto pelatihan"
+                      >
+                      <img
+                        v-else
+                        class="text-gray-500"
+                        height="22"
+                        width="22"
+                        src="@/assets/icons/IconNoImage.svg"
+                        alt="No Image"
+                      >
+                    </div>
+                  </div>
+                  <div class="registration__form-col-desc">
+                    <div class="registration__form__subtitle">
+                      Unggah foto kegiatan pelatihan yang pernah dilakukan
+                    </div>
+                    <div class="registration__form__placeholder">
+                      File yang didukung adalah .jpg, .jpeg dan .png
+                    </div>
+                    <div class="registration__form__button">
+                      <button class="registration__form__button-btn" type="button" @click="$refs.pelatihan.click()">
+                        Unggah Foto
+                        <jds-icon class="ml-2" size="12px" name="plus-bold" />
+                      </button>
+                      <input
+                        ref="pelatihan"
+                        type="file"
+                        hidden="true"
+                        accept="image/png, image/jpeg, image/svg+xml"
+                        @change="onFileChange('pelatihan')"
+                      >
+                      <div v-if="files.pelatihan.fileImage" class="registration__form__filename">
+                        Filename: {{ files.pelatihan.fileImage.get('file').name }}
+                      </div>
+                      <div v-else-if="files.pelatihan.uploadErrorMessage" class="registration__form__filename-error">
+                        {{ files.pelatihan.uploadErrorMessage }}
+                      </div>
+                      <div v-else class="registration__form__filename">
+                        Belum ada file terpilih.
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <div class="registration__submit">
+            <BaseButton class="registration__submit-btn" variant="secondary" label="Kembali" @click="onPreviousPage" />
+            <BaseButton class="registration__submit-btn" label="Selanjutnya" @click="onSubmit" />
+          </div>
+        </div>
+
+        <ModalQuestionnaire v-else :chosen-level="params.level" :village-types="villages" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import {
+  communities,
+  training,
+  villages
+} from '@/constants/questionnaire.js'
+export default {
+  data () {
+    return {
+      infoProgram: 'Program Desa Digital memiliki beberapa tingkatan/level, yuk cari tahu level desamu.',
```

Comment:
minor, this infoProgram section only in questionnaire level one #cmiiw

Reply:
😆 yes it just shows in Questionnaire lvl one